nautobot/core/tests/runner.py

```diff
@@ -179,9 +188,7 @@ def teardown_databases(self, old_config, **kwargs):
                         connection.creation.destroy_test_db(
                             suffix=str(index + 1),
                             verbosity=self.verbosity,
-                            keepdb=self.keepdb
-                            # Extra check added for Nautobot
-                            and not settings.TEST_USE_FACTORIES,
+                            keepdb=self.keepdb,
```

Comment:
Similarly, here was making it so that parallel cloned database copies were always being destroyed at the end of the test, which also appears unnecessary in my local testing.